src/bp/ui-studio/src/web/translations/en.json

```diff
@@ -381,6 +382,9 @@
         "deleteTopic": "Delete Topic",
         "deleteWorkflow": "Delete Workflow",
         "editQna": "Edit Q&A",
+        "moveQna": "Move Q&As to",
+        "confirmMoveQna": "Are you sure you want to move all Q&As to another topic?",
+        "move": "Move",
```

Comment:
Please move single word translations to ui-shared (move has already been translated)